test/clj/rems/application/test_commands.clj

```diff
@@ -1086,24 +1086,6 @@
                           :member {:name "Member Applicant 1"
                                    :email "member1@applicants.com"}}
                          injections))))
-    (testing "handler can invite members"
-      (let [application (apply-events application [{:event/type :application.event/submitted
-                                                    :event/time test-time
-                                                    :event/actor applicant-user-id
-                                                    :application/id app-id}])]
-        (is (= {:event/type :application.event/member-invited
-                :event/time test-time
-                :event/actor handler-user-id
-                :application/id app-id
-                :application/member {:name "Member Applicant 1"
-                                     :email "member1@applicants.com"}
-                :invitation/token "very-secure"}
-               (ok-command application
-                           {:type :application.command/invite-member
-                            :actor handler-user-id
-                            :member {:name "Member Applicant 1"
-                                     :email "member1@applicants.com"}}
-                           injections)))))
     (testing "other users cannot invite members"
```

Comment:
I think the original intent was probably that a handler could invite a member even before submitting. However I think that as a feature also doesn't make sense as draft should not be seen by anyone but the members. And the test used submitted state here for some weird reason.

Reply:
Yeah. Looking at the history the test has been this way for quite some time, at least since bad0a693dec3d56496438131612ce4a84e3184fd